src/App.vue

```diff
@@ -11,14 +11,25 @@
 import NavigationDrawer from '@/components/NavigationDrawer.vue';
 
 export default {
+  data(){
+    return{
+      showNav: false,
+    };
+  },
   components: {
     NavigationDrawer,
   },
+  watch:{
+  '$route.meta'(value){
+    const {hideNavigation} = value;
```

Comment:
does it work if you directly watch `hideNavigation` within the route?
```suggestion
  '$route.meta.hideNavigation'(hideNavigation){
```

Reply:
yea, this works